Cargo.toml

```diff
@@ -21,20 +21,18 @@ log = ["tracing/log"]
 default = ["tracing"]
 
 [dependencies]
-tower-service = "0.2"
+tower-service = "0.3.0-alpha.1"
 tower-load = { git = "https://github.com/tower-rs/tower.git" }
-futures = "0.1"
-tokio-executor = "0.1.5"
-tokio-sync = "0.1"
+futures-preview = "0.3.0-alpha.18"
```

Comment:
you most likely want to depend on `futures-util` and `futures-core` intead of the whole facade.